contributors/devel/sig-testing/verify-tests.md

```diff
@@ -0,0 +1,78 @@
+# Verification Tests
+
+**Table of Contents**
+
+- [Verification Tests](#verification-tests)
+  - [Overview](#overview)
+  - [`verify-govet-leveee`](#verify-govet-levee)
+
+## Overview
+
+PLACEHOLDER
+
+Verification tests for Kubernetes provide a mechanism to verify contributions for adherence to project conventions
+and best practices, and to validate generated build artifacts for soundness.
+
+Verification tests are found in vestigial shell scripts at `hack/verify-*.sh`.
+Most have been replaced by a corresponding `make` command.
+
+Most verification tests are self-explanatory.
+`verify-golint` and `verify-gofmt`, for instance, fail when a contribution does not adhere to lint and formatting conventions.
+More complex verification tests are described below.
+
+### `verify-govet-levee`
+
+Verification in `verify-govet-levee.sh` uses taint propagation analysis
+to defend against accidental logging of credentials.
+Struct fields which may contain credentials are annotated as such using the `datapolicy` field tag.
+Field tagging was introduced by [KEP-1753](https://github.com/kubernetes/enhancements/issues/1753), and analysis was introduced by [KEP-1993](https://github.com/kubernetes/enhancements/issues/1933).
+
+Taint propagation analysis defends against both direct and indirect logging of credentials.
+For consider the following hypothetical snippet
```

Comment:
I think this should just be "Consider ..." instead of "For consider ..."

Reply:
Probably iteration from `For instance, consider`.  Insufficient deletion.  Good catch.